test/model/test_lstnet.py

```diff
@@ -109,3 +107,9 @@ def test_lstnet(
         iter(tss), iter(forecasts), num_series=len(dataset.test)
     )
     assert agg_metrics["ND"] < 1.5
+
+
+if __name__ == "__main__":
+    test_lstnet(
```

Comment:
This test cast is already included in the parametrized pytest so is redundant.

Reply:
Whoops! Thanks, yes, that must be leftovers from my debugging